### PR TITLE
Update Go to 1.15 and Ubuntu to 20.04

### PR DESCRIPTION
**Pull request by masa213f**
- Update Go to 1.15
- Update Ubuntu to 20.04

`go install` outside `GOPATH` will fail with go 1.15 as follows. So I fixed the relevant part to use `go get`.
```
$ make custom-checker
if ! which custom-checker >/dev/null; then \
        cd /tmp; env GOFLAGS= GO111MODULE=on go install github.com/cybozu/neco-containers/golang/analyzer/cmd/custom-checker; \
fi
cannot find module providing package github.com/cybozu/neco-containers/golang/analyzer/cmd/custom-checker: working directory is not part of a module
Makefile:141: recipe for target 'custom-checker' failed
make: *** [custom-checker] Error 1
```

Signed-off-by: Masayuki Ishii <masa213f@gmail.com>

### CODE REVIEW

**Review by zoetrope — CHANGES_REQUESTED** [review]
Don't you update go.mod and go.sum?
Especially go.sum is incompatible.

**Comment by zoetrope** [comment]
I realized that go.sum is not included.

File: initialize/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM quay.io/cybozu/moco-mysql:8.0.20.4
 ENV GOPATH=/go
 ENV PATH=/go/bin:/usr/local/go/bin:"$PATH"
 
-ARG GO_VERSION=1.13.15
+ARG GO_VERSION=1.15.7
```
Comment (ymmt2005) [inline initialize/Dockerfile:6]: This enlarges the final image.
Why don't you split the build stage?

Comment (masa213f) [reply initialize/Dockerfile:6]: This Dockerfile is used for the unit test of the `initialize` package.
The test is launched as follows (execute `go test` in the container), so it needs to be installed.

https://github.com/cybozu-go/moco/blob/v0.3.1/Makefile#L46
```
docker run -v $(PWD):/go/src/github.com/cybozu-go/moco -e GOPATH=/tmp --rm mysql-with-go:latest sh -c "CGO_ENABLED=0 go test -v ./initialize"
```

Comment (ymmt2005) [reply initialize/Dockerfile:6]: I see.

**Review by zoetrope — APPROVED** [review]

**Pull request merged**